src/components/ContextMenu/components/MetaItem/MetaItem.props.ts

```diff
@@ -0,0 +1,42 @@
+export interface MetaItemProps {
```

Comment:
We should not have any meta item.

We can have "NotSelectable" item, but selectable state should be part of common item.

All content inside must be customised via external usage.